main/admin/user_add.php

```diff
@@ -303,10 +316,25 @@ function updateStatus(){
 $jquery_ready_content = $returnParams['jquery_ready_content'];
 
 // the $jquery_ready_content variable collects all functions that will be load in the $(document).ready javascript function
-$htmlHeadXtra[] = '<script>
+$htmlHeadXtra[] = '<script type="text/javascript">
```

Comment:
Agregar el type no es recomendado por la especificación HTML5. Ver https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script: "The HTML5 specification urges authors to omit the attribute rather than provide a redundant MIME type"